HousingSearchApi/V1/Domain/GetPersonListRequest.cs

```diff
@@ -12,9 +12,16 @@ public class GetPersonListRequest
         [MinLength(2)]
```

Comment:
Is there a particular reason to have Request and Response in the domain? I would have thought to have those outside the domain layer.

Reply:
Happy to move them elsewhere - will double check with playbook.